plugins/UserCountry/GeoIPAutoUpdater.php

```diff
@@ -268,7 +268,7 @@ public static function unzipDownloadedFile($path, $unlink = false)
             }
 
             $tempFile = GeoIp::getPathForGeoIpDatabase($tempFilename);
-            rename($tempFile, $oldDbFile);
+            copy($tempFile, $oldDbFile);
```

Comment:
Hi Richard, should we also delete the `$tempFile` then?
have you had a chance to test this works?